### ISSUE TITLE
The keyboard focus is traversing the outer-boundary of feedback window, but the keyboard focus on that window is not visible, which lead to focus loss for the feedback window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#264]
### Steps to Reproduce

1. Launch Xamarin Workbook application and select Mac platform and create a new workbook.
2. Navigate to Xamarin Workbooks-> preferences-> Hit On Enter key-> General tab is opened by default.
3. Navigate to Feedback tab-> Hit on Enter.
4. Feedback tab is opened.
5. Press on the Tab key to traverse all the interactive elements of this opened feedback tab.
6. Observe the keyboard focus next to Feedback tab item.

### Expected Behaviour
The keyboard focus is traversing the outer-boundary of the feedback window, so keyboard focus should be visible on this window, otherwise it would be also better if keyboard traversing is removed from outer boundary.

### Actual Behaviour
The keyboard focus is traversing the outer-boundary of feedback window, but that keyboard focus on that window is not visible, which lead to focus loss for the feedback window.

### User Impact:
The keyboard user might be confused due to apparent focus loss. 
![kb3_feedback_outerboundary](https://user-images.githubusercontent.com/22956130/39856988-62605466-5421-11e8-9e75-c89f0bac870b.png)

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS2.4.7
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1820
